web/templates/user_group_settings/user_group_settings.hbs

```diff
@@ -43,6 +43,21 @@
 
                 {{> group_permissions can_mention_group_widget_name="can_mention_group" can_manage_group_widget_name="can_manage_group"}}
             </div>
+
+            <div class="group_detail_box">
+                <div class="user_group_details_box_header">
+                    <h3 class="user_group_setting_subsection_title">
+                        {{t "User group details" }}
+                    </h3>
+                </div>
+                <div class="creator_details group_detail_box">
```

Comment:
This `group_detail_box` class inside `group_detail_box` class seems weird. Would be better to have different class names like we have for streams.